target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -13,6 +13,11 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(mtd_get_mac_binary art 0x12)
 		;;
+	dongwon,dw02-412h-64m|\
+	dongwon,dw02-412h-128m)
+		caldata_extract "art" 0x5000 0x844
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) +4)
```

Comment:
no address in 0x5006?

Reply:
Yes, the values in that location are the same for all devices.